extensions/amp-story/1.0/amp-story-quiz.js

```diff
@@ -253,12 +270,41 @@ export class AmpStoryQuiz extends AMP.BaseElement {
   }
 
   /**
-   * Triggers changes to quiz state on response interaction
+   * Triggers the analytics event for quiz response.
+   *
+   * @param {!Element} optionEl
+   * @private
+   */
+  triggerAnalytics_(optionEl) {
+    this.variableService_.onVariableUpdate(
+      AnalyticsVariable.INTERACTIVE_ELEMENT_ID,
+      this.element.getAttribute('id')
+    );
+    this.variableService_.onVariableUpdate(
+      AnalyticsVariable.INTERACTIVE_ELEMENT_RESPONSE,
+      optionEl.optionId_
+    );
+    this.variableService_.onVariableUpdate(
```

Comment:
The interactive_element_type is introduced by this PR. So it would not be another value different from `quiz`? When will we set it back to null if one stops interacting with the quiz?

Reply:
same applies to the other two variables.

Reply:
I can reset the variables to null once the analytics event has been triggered

Reply:
Update: I reverted the change resetting the variables to null, as none of the other variables are reset in this manner and it makes it difficult to test the variable values